sdks/python/apache_beam/runners/interactive/extensions/apache-beam-jupyterlab-sidepanel/src/SidePanel.ts

```diff
@@ -41,30 +41,20 @@ export class SidePanel extends BoxPanel {
   constructor(
     manager: ServiceManager.IManager,
     rendermime: IRenderMimeRegistry,
-    pageType: string
+    sessionContext: SessionContext,
+    title: string,
+    widget: ClustersWidget | InteractiveInspectorWidget
```

Comment:
You can replace the type here with "ReactWidget" b/c both the ClustersWidget and InteractiveInspectorWidget extend this.

Reply:
That would be better and it would certainly reduce clutter as well. Thanks!